docs/linux/packaging.md

```diff
@@ -18,6 +18,84 @@ intended as a staging environment for testing. Packages on [pso](http://packages
 are uploaded manually after the packages received some testing. End users usually have only
 pso enabled.
 
+## Building packages locally
+
+It is possible to build packages locally. This can be done either directly on the machine,
+or in a Docker container which is more similar to the CI environment.
+
+### Building packages locally directly on the machine
+
+To build packages locally, you'll have to have the dependencies installed.
+
+The easiest way to achieve this is to use the `mk-build-deps` tool:
+
+```bash
+sudo apt install dh-python python3-all debhelper help2man devscripts \
+  equivs jq quilt
+sudo mk-build-deps --install linux/debian/control
+```
+
+Then you can build the Debian package with:
+
+```bash
+KEYMAN_TIER=$(cat TIER.md)
+export KEYMAN_TIER
+cd linux
+./scripts/deb-packaging.sh source
+mkdir -p make_deb
+cd make_deb
+dpkg-source --extract ../../keyman_*.dsc
+cd keyman-*
+dch -v$(cat VERSION.md)-1 ""
+debuild -us -uc
+```
```

Comment:
Is this the same as in keyman-config.md? Are they likely to diverge in the future?